### DISCUSSION TITLE
Writing Visual Studio Extension (VSIX) with .NET 6.0

### DISCUSSION THREAD

**Discussion by cjdutoit** [VsixCommunity/Community.VisualStudio.Toolkit#331] [Q&A] [answered]
@madskristensen / community do you know  if updating the extension templates from .Net Framework 4.8 to net6.0 is on the roadmap?   Would be great to see these templates updated so we can use the newer language features as well as consume newer packages.

**Comment by StevenRasmussen** [comment, marked as answer]
Please see #235, #233, and #56.

**Reply by cjdutoit** [reply]
Thanks
